src/Microsoft.DotNet.Build.Tasks/PackageFiles/versioning.targets

```diff
@@ -64,12 +66,12 @@
       <AssemblyInfoLines Include="[assembly:AssemblyTitle(&quot;$(AssemblyName)&quot;)]" />
       <AssemblyInfoLines Include="[assembly:AssemblyDescription(&quot;$(AssemblyName)&quot;)]" />
```

Comment:
Shouldn't `assembly:AssemblyDescription` use the `$(Description)` property above?